src/list-data/request-action-mapping.ts

```diff
@@ -76,16 +76,16 @@ export const XproMapping: RequestActionMappingI = {
   REH: [
     EntityTypes.RLC, // maps to Corp Type "LLC"
     EntityTypes.XCP, // maps to Corp Type "XCP"
-    EntityTypes.XSO, // maps to Corp Type "XS"
-    EntityTypes.XUL // maps to Corp Type "A"
+    EntityTypes.XCR, // maps to Corp Type "EXTRA_PRO_A"
```

Comment:
1. XCR === A

2. I sorted this list.

3. I removed XUL from below as per original NR code (before WoN changes).